rom/laghos_timeinteg.cpp

```diff
@@ -50,6 +50,8 @@ void RK2AvgSolver::Step(Vector &S, double &t, double &dt)
         return;
     }
 
+    RKStages.clear();
```

Comment:
It is unclear whether this would have a significant performance impact, but clearing the data and pushing back vectors on each timestep is less efficient than keeping data allocated throughout the simulation and copying. That is, the constructor for the solver can allocate as many stages as needed in `RKStages`, and then on each stage just copy `S` to `RKStages[i]`.

Reply:
Resolved.